nrf802154_sniffer/nrf802154_sniffer.py

```diff
@@ -151,6 +151,17 @@ def correct_time(self, sniffer_timestamp):
 
             return sniffer_timestamp + overflow_count * self.TIMER_MAX
 
+    def update_channel(self, channel):
```

Comment:
I'm not a fan of current name either. `set_channel` was my first idea, but it sounds like a typical setter. This method is more than a setter - it is used to update the channel value after the channel was set during initialization. That's how I came up with `update`. Do you think that `set_channel` would be better anyway?

Reply:
Yes, IMO `set_channel` would sound better. I don't think that it requires a special name just because it changes previously initialized value.
One could also use `@property` decorator, but I think it would make it less readable here.